### ISSUE TITLE
localStorage changes based on intel address

### ISSUE THREAD

**Issue by nhamer** [iitc-project/ingress-intel-total-conversion#1226]
This isn't a bug in IITC, but it is an issue - if a user logs in to www.ingress.com IITC sees one set of localStorage. If the user subsequently logs in to ingress.com, IITC sees a new, blank localStorage and all preferences are reset. 

Versions of intel prior to Oct, 2016 redirected ingress.com/intel to www.ingress.com/intel, so whichever site you logged into, you got a consistent view of localStorage.

Could be fixed with something similar to force-https.user.js (which incidentally is obsolete)

**Comment by AlterTobi** [comment]
I'd prefer to set an browser bookmark instead.

I've created a new plugin to force www in hostname.

**Comment by nhamer** [comment]
Yeah, the gotcha is if someone sends you intel links to portals, they'll contain whichever version they originated them with, and darn your bookmark
